anaconda.spec.in

```diff
@@ -182,6 +182,11 @@ BuildRequires: desktop-file-utils
 Requires: anaconda-gui = %{version}-%{release}
 Requires: zenity
 Recommends: xhost
+# FIXME: This is currently needed by the locale1-x11-sync script.
+# It makes little sense for Anaconda to pull in two Python DBus libraries,
+# so we should either split the script to a separate or change it to also
+# use dasbus like the rest of Anaconda.
+Requires: python3-dbus-next
```

Comment:
Maybe split it as a subpackage in this specfile, that way it's easy to move to an independent package later.

Reply:
That is a good point.

If we go this way, we would have to do a side tag build between kiwi/fedora-kickstarts and anaconda or we would have to remove it first from Anaconda which would break stuff. Is there a better way?

On the other hand if this will be a new subpackage then we can exclude this sub-package in kiwi/fedora-kickstarts with the same change-set as adding a new one. However, I wonder how to reasonably include as dependency of anaconda-live, did the weak deps works as expected? I know lorax ignores weak deps but live is build with Anaconda so it might be fine?

Any recommendation how to handle the switch @Conan-Kudo?

Reply:
Do it as a subpackage, weak deps are honored in both kiwi and lorax.

Reply:
This new subpackage would have `Supplements: (xorg-x11-server-Xorg and livesys-scripts)` and would be automatically pulled in accordingly.

Reply:
I'm pretty sure that Lorax ignores weak deps, but not for Live because there is Anaconda who installs those packages.

Anyway, we can move it is separate sub-package later, that should be easy operation and currently we have our hands full. Let's go with this version and we can fine-tune it later.

Reply:
But we don't use non-live lorax for X11 environments, so it's not an issue.